src/main/webapp/difficulty.csv

```diff
@@ -0,0 +1,100 @@
+6

```

Comment:
just something to note - do you expect students to enter difficulty on a sliding scale or freehand in a form? if it's the second, then you might want to add some strings (non-numbers basically) to this .csv and come up with a way to handle them. since you're using parseInt, I think (?) it would crash on anything that wasn't an int.

Reply:
csv are actually all string based, that is why I parse them an integers so I don't think it will crash anything.

Reply:
I meant if it's open-ended and someone puts in "hello" or another symbol on accident instead of a number - probably isn't something you need to worry about now though haha

Reply:
Good thing to think about. Ideally, I think, your frontend would only allow numbers to be sent (which the server would then verify before storing them). This is good for now though

Reply:
Got it. I can handle that in code or we could make the input interface numbers only.